improving tools readme (#20305)
commaai/openpilot: tools/sim/README.md

To see the options for changing the environment, such as the town, spawn point or precipitation, you can run `./start_openpilot_docker.sh --help`.
This will print the help output inside the docker container. You need to exit the docker container before running `./start_openpilot_docker.sh` again.

## Low quality mode

It is possible that you may run into performance issues with CARLA because of reasons stated in [this](https://github.com/carla-simulator/carla/issues/2617) thread. For this case we have a low quality mode you can activate by running:
```

> ./start_openpilot_docker.sh --low_quality=True

Reviewer: shouldn't have to specify true for this arg, should just be based on presence

To see the options for changing the environment, such as the town, spawn point or precipitation, you can run `./start_openpilot_docker.sh --help`.
This will print the help output inside the docker container. You need to exit the docker container before running `./start_openpilot_docker.sh` again.

## Performance

> openpilot doesn't have any extreme hardware requirements, however CARLA is very resource-intensive and might cause issues for people using lower-end systems. For this case we have a low quality mode you can activate by running:

Reviewer: ```suggestion
openpilot doesn't have any extreme hardware requirements, however CARLA is very resource-intensive and may not run smoothly on your system. For this case, we have a low quality mode you can activate by running:
```

> openpilot doesn't have any extreme hardware requirements, however CARLA is very resource-intensive and might cause issues for people using lower-end systems. For this case we have a low quality mode you can activate by running:
> ```
> ./start_openpilot_docker.sh --low_quality
> ```
> You can also check out the [CARLA python documentation](https://carla.readthedocs.io/en/latest/python_api/) to find more parameters to tune that might increase performance on your system

Reviewer: ```suggestion
openpilot doesn't have any extreme hardware requirements, however CARLA is very resource-intensive and may not run smoothly on your system. For this case, we have a low quality mode you can activate by running:
```
./start_openpilot_docker.sh --low_quality
```
You can also check out the CARLA [python documentation](https://carla.readthedocs.io/en/latest/python_api/) to find more parameters to tune.
```
